bin/iabc.js

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env node
+import make from '../src/cli/make.js';
+import harCreate from '../src/cli/har-route-builder.js';
+import openApiCreate from '../src/cli/open-api-route-builder.js';
+import curlCreate from '../src/cli/curl-route-builder.js';
+
+//get command from command line arguments
+const command = process.argv[2];
```

Comment:
please get the logic into a ts file, so we can test it :) sorry about the late reaction